src/app/components/Footer/index.tsx

```diff
@@ -23,6 +26,37 @@ export const Footer = memo(() => {
   const walletHasAccounts = useSelector(selectHasAccounts)
   const isMobile = React.useContext(ResponsiveContext) === 'small'
   const { t } = useTranslation()
+  const [showCommitLink, setShowCommitLink] = useState(false)
+
+  const toggleCommitLink = () => {
+    setShowCommitLink(!showCommitLink)
+  }
+
+  function CommitLink() {
```

Comment:
I would keep it inline as it's not reusable.

Reply:
Good point, I have moved it inline